src/vm/comthreadpool.cpp

```diff
@@ -678,6 +710,19 @@ void WINAPI BindIoCompletionCallbackStub(DWORD ErrorCode,
 #endif // !FEATURE_PAL
 }
 
+FCIMPL1(VOID, ThreadPoolNative::TraceAllocateNativeOverlapped, LPOVERLAPPED lpOverlapped)
+{
+    FCALL_CONTRACT;
+
+    HELPER_METHOD_FRAME_BEGIN_RET_0();
+
+    if (ETW_EVENT_ENABLED(MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER_Context, ThreadPoolIODequeue))
```

Comment:
Check of this condition can be before the `HELPER_METHOD_FRAME_BEGIN` to minimize overhead when the tracing is off.

Reply:
Wanted to ask about that.